resources/src/asset.rs

```diff
@@ -85,6 +85,15 @@ impl AssetIdentifier {
         }
     }
 
+    /// The code of this asset as a result.
+    pub fn asset_code(&self) -> Option<String> {
```

Comment:
Interesting. This would have been useful for the trades I believe as well. I wonder if there's another name that is more indicative that this will be optional. Trying to understand `code` for `asset_code`